README.md

```diff
@@ -497,6 +497,7 @@ And we're back! Use this repo to share and keep track of software, tech, CS, PM,
 | [Yahoo](https://ouryahoo.wd5.myworkdayjobs.com/en-US/careers/job/United-States-of-America---Remote/Software-Engineering-Intern_JR0021366) | Remote (US) | Software Engineering Intern |
 | [Dayton Freight Lines, Inc](https://careers-daytonfreight.icims.com/jobs/7695/software-developer-intern/job?mode=job&iis=SOURCE&iisn=LinkedIn&mobile=false&width=1296&height=500&bga=true&needsRedirect=false&jan1offset=-480&jun1offset=-420) | Dayton, OH | Software Developer Intern |
 | [Gusto](https://boards.greenhouse.io/gusto/jobs/4645583) | SF, Denver, NYC | |
+| [Ampere computing](https://amperecomputing.com/apply/?p=search%3Fc%3D%26l%3D%26q%3D) | Santa Clara | Solutions Engineering Intern | 
```

Comment:
Where is the Solutions Engineering Intern application? Is it the general application?

This isn't clear to me. Ideally, you should provide a link to a specific job application page (e.g., https://amperecomputing.com/apply/?p=job%2FoXmVkfwi, which is for the Technical Intern application page).

Reply:
It seems that the website doesn't preserve URL-encoded search parameters. I inputted intern in the search box and it was one of the positions listed there (along with other computer engineering positions that I thought might interest anyone). I'll replace it with a specific position link.